pkg/nsx/services/common/types.go

```diff
@@ -90,6 +90,7 @@ const (
 	AnnotationAttachmentRef            string = "nsx.vmware.com/attachment_ref"
 	AnnotationRestore                  string = "nsx/restore"
 	LabelCPVM                          string = "iaas.vmware.com/is-cpvm-subnetport"
+	AnnotationAssociatedResource       string = "nsx.vmware.com/associated_resource"
```

Comment:
better move it under other annotations having the same format, i.e. under AnnotationAttachmentRef

Reply:
right，moved